daemon/kmd/lib/kmdapi/responses.go

```diff
@@ -54,95 +54,194 @@ type VersionsResponse struct {
 	Versions []string `json:"versions"`
 }
 
+// Response to `GET /versions`
+// swagger:response VersionsResponse
+type versionsResponse struct {
+	//Versions Response
+	//in:body
+	Body *VersionsResponse
+}
+
 // APIV1GETWalletsResponse is the response to `GET /v1/wallets`
 // friendly:ListWalletsResponse
 type APIV1GETWalletsResponse struct {
 	APIV1ResponseEnvelope
 	Wallets []APIV1Wallet `json:"wallets"`
 }
 
+// Response to `GET /v1/wallets`
+// swagger:response ListWalletsResponse
+type listWalletsResponse struct {
+	//in: body
+	Body *APIV1GETWalletsResponse
+}
+
 // APIV1POSTWalletResponse is the response to `POST /v1/wallet`
 // friendly:CreateWalletResponse
 type APIV1POSTWalletResponse struct {
 	APIV1ResponseEnvelope
 	Wallet APIV1Wallet `json:"wallet"`
 }
 
+// Response to `POST /v1/wallet`
+// swagger:response CreateWalletResponse
+type createWalletResponse struct {
+	//	in:body
+	Body *APIV1POSTWalletResponse
+}
+
 // APIV1POSTWalletInitResponse is the response to `POST /v1/wallet/init`
 // friendly:InitWalletHandleTokenResponse
 type APIV1POSTWalletInitResponse struct {
 	APIV1ResponseEnvelope
 	WalletHandleToken string `json:"wallet_handle_token"`
 }
 
+// Response to `POST /v1/wallet/init`
+// swagger:response InitWalletHandleTokenResponse
+type initWalletHandleTokenResponse struct {
+	//	in:body
+	Body *APIV1POSTWalletInitResponse
+}
+
 // APIV1POSTWalletReleaseResponse is the response to `POST /v1/wallet/release`
 // friendly:ReleaseWalletHandleTokenResponse
 type APIV1POSTWalletReleaseResponse struct {
 	APIV1ResponseEnvelope
 }
 
+// Response to `POST /v1/wallet/release`
+// swagger:response ReleaseWalletHandleTokenResponse
+type releaseWalletHandleTokenResponse struct {
+	//	in:body
+	Body *APIV1POSTWalletReleaseResponse
+}
+
 // APIV1POSTWalletRenewResponse is the response to `POST /v1/wallet/renew`
 // friendly:RenewWalletHandleTokenResponse
 type APIV1POSTWalletRenewResponse struct {
 	APIV1ResponseEnvelope
 	WalletHandle APIV1WalletHandle `json:"wallet_handle"`
 }
 
+// Response `POST /v1/wallet/renew`
+// swagger:response RenewWalletHandleTokenResponse
+type renewWalletHandleTokenResponse struct {
+	//	in:body
+	Body *APIV1POSTWalletRenewResponse
+}
+
 // APIV1POSTWalletRenameResponse is the response to `POST /v1/wallet/rename`
 // friendly:RenameWalletResponse
 type APIV1POSTWalletRenameResponse struct {
 	APIV1ResponseEnvelope
 	Wallet APIV1Wallet `json:"wallet"`
 }
 
+// Response to `POST /v1/wallet/rename`
+// swagger:response RenameWalletResponse
+type renameWalletResponse struct {
+	//	in:body
+	Body *APIV1POSTWalletRenameResponse
+}
+
 // APIV1POSTWalletInfoResponse is the response to `POST /v1/wallet/info`
 // friendly:WalletInfoResponse
 type APIV1POSTWalletInfoResponse struct {
 	APIV1ResponseEnvelope
 	WalletHandle APIV1WalletHandle `json:"wallet_handle"`
 }
 
+// Response to `POST /v1/wallet/rename`
```

Comment:
rename -> info ?

Reply:
the typo is fixed. I also put in a quick fix for the first warning.  I see that swagger:ignore annotation was already added for for APIV1RequestEnvelope but the definition still gets created in swagger.json. I'm unable to fix this warning at this time. swagger.json is still valid with this redundant definition.